### ISSUE TITLE
Check failed: height <= datum_height (224 vs. 214)

### ISSUE THREAD

**Issue by erichouyi** [yjxiong/caffe#84]
Hi @wanglimin @yjxiong 
Recently, i want to do finetuning on ucf11 dataset (320 x 240) with "cuhk_action_temporal_vgg_16_split1.caffemodel". AS i only have one GPU, so i modify "vgg_16_flow_train_val_fast.prototxt". To be specific,i  remove Gather layers, and restore all blob names with by "_local" to their original names with out the suffix. Besides, i rename "fc8" to "fc8-1", and the "num_output" is 11.
However, i have the following problem:

I0615 10:58:27.396116 21338 solver.cpp:47] Solver scaffolding done.
I0615 10:58:27.396193 21338 caffe.cpp:86] Finetuning from models/action_recognition/cuhk_action_temporal_vgg_16_split1.caffemodel
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:505] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:78] The total number of bytes read was 538739211
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:505] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:78] The total number of bytes read was 538739211
I0615 10:58:29.340661 21338 solver.cpp:363] Solving Action_TemporalNet
I0615 10:58:29.340751 21338 solver.cpp:364] Learning Rate Policy: step
F0615 10:58:31.520954 21351 data_transformer.cpp:313] Check failed: height <= datum_height (224 vs. 214) 
**\* Check failure stack trace: ***
    @     0x7f19d1005daa  (unknown)
    @     0x7f19d1005ce4  (unknown)
    @     0x7f19d10056e6  (unknown)
    @     0x7f19d1008687  (unknown)
    @     0x7f19d13970e0  caffe::DataTransformer<>::Transform()
    @     0x7f19d141cd9d  caffe::VideoDataLayer<>::InternalThreadEntry()
    @     0x7f19c9403a4a  (unknown)
    @     0x7f19c8cbb182  start_thread
    @     0x7f19d05f04ad  (unknown)
    @              (nil)  (unknown)
Aborted (core dumped)

could you tell me how to solve it ?

**Comment by yjxiong** [comment]
One of your frames has a dimension less than 224 (214 as indicated). Please make sure they are larger than 224x224.

**Comment by erichouyi** [comment]
@yjxiong  thank you very much! I solve the problem, i find some frame be 320 x 216 , and the height is less than 224. After removing these frame, the problem does not appear.

**Comment by yjxiong** [comment]
Good to know. Then closing this issue.
